python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -303,6 +315,15 @@ def sample_neighbors(
             corresponding to each neighboring edge of a node. It must be a 1D
             floating-point or boolean tensor, with the number of elements
             equalling the total number of edges.
+        return_eids: bool
+            Boolean indicating whether the edge IDs of sampled edges, should be
```

Comment:
Do we always have eids as edge attribute in CSCSamplingGraph?

Reply:
No, so the pre-check should check that.

Reply:
We should mention it here in the documentation, say it works only if eids is edge attribute.

Meanwhile, we need to ask users to specify whether the eids should be added as edge attribute in the preprocess or not.